languages/english/pack/src/main/res/xml/english_keyboards.xml

```diff
@@ -43,5 +43,9 @@
               layoutResId="@xml/terminal" id="b1c24b40-02ce-4857-9fb8-fb9e4e3b4318"
               description="@string/terminal_keyboard_description"
               index="9"/>
+    <Keyboard nameResId="@string/halmak_keyboard"
```

Comment:
I notice now that this keyboard does not have `defaultDictionaryLocale="en"`
This means that suggestions will not be on by default for it. Was that on purpose?